invenio_app_ils/records/jsonresolver/circulation.py

```diff
@@ -31,7 +31,7 @@ def _circulation_for_document_resolver(document_pid):
 
         for pid in item_pids:
             items_count += 1
-            if not is_item_available(pid):
+            if not is_item_available(pid) and circulation_is_item_in_loanable_state(pid):
```

Comment:
Just to note that I have change this method in my PR, so better if you merge it before me and I will resolve the conflict.